app/models/chef.rb

```diff
@@ -1,4 +1,8 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+
+   def unique_ingredients
+      dishes.joins(:ingredients).pluck('ingredients.name').uniq
+   end
```

Comment:
The .uniq is technically Ruby, I'd rather see the .distinct. The difference being that with .uniq, AR is pulling ALL the ingredients then going through them and getting rid of dups whereas .distinct only pulls distinct records from the db. 
Otherwise, this looks good